webapp/channels/src/sass/layout/_markdown.scss

```diff
@@ -41,8 +41,6 @@ h6.markdown__heading {
 }
 
 .markdown__paragraph-inline {
-    display: inline;
```

Comment:
Why is this needed? How will this change how other things are displayed as of today?